### ISSUE TITLE
All workers need decent doc strings (100 cow points)

### ISSUE THREAD

**Issue by brifordwylie** [SuperCowPowers/old-workbench-archived#103]
Mostly mind numbing but put a bit of thought into this.. perhaps this kind of docstring:

```
    ''' Store a sample into the DataStore.
        Args:
            filename: name of the file (used purely as meta data not for lookup)
            input_bytes: the actual bytes of the sample e.g. f.read()
            type_tag: ('pe','pcap','pdf','json','swf', or ...)
        Returns:
            the md5 of the sample
    '''
```

But the the case of a worker 'Args' turns into 'Dependencies' and 'Returns' turns into 'Output'

**Comment by brifordwylie** [comment]
The snippet above is based on google guide http://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Comments#Comments

**Comment by brifordwylie** [comment]
moving this to 0.3 release so we can focus on repo-reorg

**Comment by iamkhush** [comment]
We need to have sphinx napoleon extension for this, but its available in 1.3 . We need to upgrade sphinx to 1.3 . http://sphinx-doc.org/latest/ext/napoleon.html

**Comment by brifordwylie** [comment]
hmm..when I look at https://pypi.python.org/simple/Sphinx/  I don't see 1.3... so does that mean we have to wait on this until 1.3 is out? (which is fine, having the docstrings nice will be good no matter what)
